src/Console/Commands/MvcContractUpdate.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Tychovbh\Mvc\Console\Commands;
+
+use Illuminate\Console\Command;
+use Tychovbh\Mvc\Contract;
+use Tychovbh\Mvc\Services\DocumentSign\DocumentSignInterface;
+
+class MvcContractUpdate extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'mvc-contract:update';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'This command updates the signed_at from a contract, when the status is si(signed)';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @param DocumentSignInterface $documentSign
+     * @return mixed
+     */
+    public function handle(DocumentSignInterface $documentSign)
+    {
+        try {
+            $contracts = Contract::whereNotNull('external_id')->whereNull('signed_at')->get();
+
+            foreach ($contracts as $contract) {
+                $document = $documentSign->show($contract['external_id']);
```

Comment:
this might trigger exception. The try catch should be here.

Now if one contract document show request failes, all following contracts will be stopped from updating